packages/block-library/src/page-list/index.php

```diff
@@ -192,7 +192,8 @@ function block_core_page_list_render_nested_page_list( $open_submenus_on_click,
 			$css_class .= ' menu-item-home';
 		}
 
-		$title      = wp_kses_post( $page['title'] );
+		$title = wp_kses_post( $page['title'] ) ? wp_kses_post( $page['title'] ) : __( '(no title)' );
```

Comment:
The `wp_kses_post` can be expensive. Maybe we should run it just once. What do you think?

```php
$title =  wp_kses_post( $page['title'] );
$title =  $title ? $title :  __( '(no title)' );
```

P.S. I've not tested this code :)